### PR TITLE
dz60rgb_wkl move features, rgb_matrix to info.json

### PR DESCRIPTION
**Pull request by haunt98**
dz60rgb_wkl move features, rgb_matrix from rules.mk, config.h to info.json thanks to data driven config.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/dztech/dz60rgb_wkl/v1/rules.mk [hunk 1]
```diff
@@ -1,14 +1,4 @@
...
-MOUSEKEY_ENABLE = yes          # Mouse keys
-EXTRAKEY_ENABLE = yes          # Audio control and System control
-CONSOLE_ENABLE = no            # Console for debug
-COMMAND_ENABLE = no            # Commands for debug and configuration
-NKRO_ENABLE = yes           # Enable N-Key Rollover
-BACKLIGHT_ENABLE = no          # Enable keyboard backlight functionality
-RGBLIGHT_ENABLE = no           # Enable keyboard RGB underglow
-AUDIO_ENABLE = no              # Audio output
-RGB_MATRIX_ENABLE = yes        # Use RGB matrix
 NO_USB_STARTUP_CHECK = no      # Disable initialization only when usb is plugged in
```
Comment (dunk2k) [inline keyboards/dztech/dz60rgb_wkl/v1/rules.mk:4]: could this setting be migrated to rev1/info.json (`usb.no_startup_check=true`)

File: keyboards/dztech/dz60rgb_wkl/v2/rules.mk [hunk 2]
```diff
@@ -1,14 +1,4 @@
...
-MOUSEKEY_ENABLE = yes          # Mouse keys
-EXTRAKEY_ENABLE = yes          # Audio control and System control
-CONSOLE_ENABLE = no            # Console for debug
-COMMAND_ENABLE = no            # Commands for debug and configuration
-NKRO_ENABLE = yes           # Enable N-Key Rollover
-BACKLIGHT_ENABLE = no          # Enable keyboard backlight functionality
-RGBLIGHT_ENABLE = no           # Enable keyboard RGB underglow
-AUDIO_ENABLE = no              # Audio output
-RGB_MATRIX_ENABLE = yes        # Use RGB matrix
 NO_USB_STARTUP_CHECK = no      # Disable initialization only when usb is plugged in
```
Comment (dunk2k) [inline keyboards/dztech/dz60rgb_wkl/v2/rules.mk:4]: could this setting be migrated to rev2/info.json (`usb.no_startup_check=true`)

**Review by dunk2k — CHANGES_REQUESTED** [review]

**Comment by dunk2k** [comment]
Would you be able to address linting check failures?

File: keyboards/dztech/dz60rgb_wkl/v1/rules.mk (hunk 1 above)
Comment (haunt98) [reply keyboards/dztech/dz60rgb_wkl/v1/rules.mk]: Thanks @dunk2k, I don't know about that. Just push an update following your suggestion.

File: keyboards/dztech/dz60rgb_wkl/v2/rules.mk (hunk 2 above)
Comment (haunt98) [reply keyboards/dztech/dz60rgb_wkl/v2/rules.mk]: As above comment, I push an update to fix this.

**Pull request closed without merging**